run_nwaku.sh

```diff
@@ -36,14 +81,15 @@ exec /usr/bin/wakunode\
       --rest-private=true\
       --rest-address=0.0.0.0\
       --rln-relay=true\
-      --rln-relay-dynamic=false\
-      --rln-relay-membership-index=${NODE_INDEX}\
+      --rln-relay-dynamic=true\
+      --rln-relay-eth-client-address="$RPC_URL"\
+      --rln-relay-eth-contract-address=$RLN_CONTRACT_ADDRESS\
       --dns-discovery=true\
```

Comment:
missing --rln-relay-cred-path ? otherwise it will try to find the keystore in a default location.

Reply:
added in commit [baef6d5](https://github.com/waku-org/waku-simulator/pull/23/commits/baef6d54314c9e4bb300b2973c8d34761c44f48f)